README.md

```diff
@@ -69,7 +69,9 @@ Currently, [Azure Key Vault plugin for Notation](https://github.com/Azure/notati
     plugin_url: https://github.com/Azure/notation-azure-kv/releases/download/v1.0.1/notation-azure-kv_1.0.1_linux_amd64.tar.gz
```

Comment:
We may also need to update this AKV plugin to the latest version `v1.2.0`: https://github.com/Azure/notation-azure-kv/releases/tag/v1.2.0

Reply:
This is actually fine? Because it's just an example and independent of any vendor plugin releases. But yes, why not update to v1.2.0.